.github/workflows/build-complete-samples.yml

```diff
@@ -843,6 +843,10 @@ jobs:
               name: 'tab-navbar-menu'
               version: '16.x'
 
+            - project_path: 'samples/bot-types-of-cards/nodejs'
```

Comment:
Kindly create deployment pipeline as well and push in this PR.

Reply:
We are not creating deployment pipeline for nodejs i think, when sample exists in Csharp.